deploy/yamls/ubuntu/zfs-driver.yaml

```diff
@@ -829,6 +877,50 @@ spec:
           hostPath:
             path: /lib/libnvpair.so.1.0.1
             type: File
+        - name: exportfs
+          hostPath:
+            path: /usr/sbin/exportfs
+            type: File
+        - name: mount-nfs
+          hostPath:
+            path: /usr/sbin/mount.nfs
+            type: File
+        - name: rpc-nfsd
+          hostPath:
+            path: /usr/sbin/rpc.nfsd
```

Comment:
why we need this?

Reply:
I do not find out the "exact" path how sharing and unsharing datasets needs to be done to free the kernel locks after a dataset has been shared.

I did the whole bunch for loss of better knowledge of zfs share and nfs-kernel interaction using trial and error.

I left that alone when I got the unsharing and destroying to work. I am unsure if it is really needed. Considering that I still observe that it sometimes takes up to five minutes to get the share mounted on my VirtualBoxen, I'd like to leave the task "derive the minimal set of necessary binaries"  as one of the last steps.  Do you know more about the zfs and in kernel NFS interaction?